tensorflow_estimator/python/estimator/run_config.py

```diff
@@ -293,7 +293,7 @@ def _validate(property_name, cond, message):
                     ' one argument "op".')
 
   _validate('protocol',
-            lambda protocol: protocol in (None, "grpc", "grpc+verbs"),
+            lambda protocol: protocol in (None, "grpc", "grpc+gdr", "grpc+verbs"),
             message='protocol should be grpc or grpc+verbs')
```

Comment:
Update the message too?

Reply:
@karmel  This seems fine to me.

Reply:
@byronyi byron, would you pls follow this issue for me? Because "grpc+gdr" is implemented by you, it's better for you to discuss the design of upper layer caller API. Thanks, byron!!! :-)